src/accelerate/commands/env.py

```diff
@@ -55,6 +56,7 @@ def env_command(args):
         "Python version": platform.python_version(),
         "Numpy version": np.__version__,
         "PyTorch version (GPU?)": f"{pt_version} ({pt_cuda_available})",
+        "PyTorch version (GPU?)": f"{pt_version} ({pt_xpu_available})",
```

Comment:
```suggestion
        "PyTorch XPU available": str(pt_xpu_available),
```
No need to repeat the version here.